test/select.spec.js

```diff
@@ -1139,7 +1140,7 @@ describe('ui-select tests', function () {
       var el = createUiSelect();
       expect(el.find('.ui-select-choices-group .ui-select-choices-group-label').map(function () {
         return this.textContent;
-      }).toArray()).toEqual(["Foo", "Baz", "bar"]);
```

Comment:
Interesting that this line changed.

Reply:
Yeah that is weird... @LukaszKokot do you know why this changed?